apollo-portal/pom.xml

```diff
@@ -30,6 +30,20 @@
 		<github.path>${project.artifactId}</github.path>
 		<maven.build.timestamp.format>yyyyMMddHHmmss</maven.build.timestamp.format>
 	</properties>
+
+	<dependencyManagement>
```

Comment:
Is it better to move this dependency management section to parent pom?

Reply:
Yes, I agree